clients/mobile/src/authentication/services/authService.ts

```diff
@@ -123,17 +127,21 @@ export const loginUser = async(email: string, password: string): Promise<boolean
   }
 }
 
-export const googleLogin = async(): Promise<void> => {
+export const googleLogin = async(): Promise<boolean> => {
```

Comment:
i am not sure why this has to return anything? is this an action? why are we returning true false? if we failed to login shouldn't we throw? thats what you've done in most places

Reply:
Again, it is back to its previous state and returning void. If it fails then it will throw an Error. [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/239/commits/4356c490c82c28f3876f0b0d21519b1caaff1826)